Packs/SIGNL4/Integrations/SIGNL4/SIGNL4_description.md

```diff
@@ -0,0 +1,3 @@
+In order to configure the SIGNL4 pack in Cortex XSOAR you just need the SIGNL4 team or integration secret (Password). You can find the same in your SIGNL4 web portal under Teams or under Integrations -> Installed -> Inbound Webhook. It is the last part of your webhook URL.
+
+You find more information about SIGNL4 at [signl4.com](https://signl4.com).
```

Comment:
```suggestion
You find more information about SIGNL4 at [signl4.com](https://signl4.com).

Note: This is a beta Integration, which lets you implement and test pre-release software. Since the integration is beta, it might contain bugs. Updates to the integration during the beta phase might include non-backward compatible features. We appreciate your feedback on the quality and usability of the integration to help us identify issues, fix them, and continually improve.
```